src/client/app/components/meters/EditMeterModalComponent.tsx

```diff
@@ -180,6 +180,34 @@ export default function EditMeterModalComponent(props: EditMeterModalComponentPr
 		setLocalMeterEdits({ ...localMeterEdits, [e.target.name]: JSON.parse(e.target.value) });
 	};
 
+	// Function handles the selection of a new displayable.
+	const handleDisplayableChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+		// If there is a potential issue then the admin will decide if save happens. Otherwise, the value is put into state.
+		let save = true;
+		if (!JSON.parse(e.target.value)) {
+			// This will hold the overall message for the admin alert.
+			let msg = '';
+			// This will hold the names of groups that are affected.
+			let groups = '';
+			// Tells if the change should be cancelled.
+			// Checks for groups that include the meter being edited.
+			for (const groupId of Object.values(groupDataByID)) {
+				if (groupId.deepMeters.includes(meterState.id)) {
```

Comment:
This is my bad for not thinking of this earlier. If the group has its displayable as no/false then it should not be listed and should not impact saving the change. The logic is that if you cannot see the group then hiding a meter with that group does not change the situation.